test/main.js

```diff
@@ -1,11 +1,10 @@
 'use strict';
 
+var concatStream = require('concat-stream');
 var replacePlugin = require('../');
 var fs = require('fs');
-var es = require('event-stream');
 var should = require('should');
 var File = require('vinyl');
-require('mocha');
```

Comment:
Yes, `mocha` command automatically sets `describe`, `it` etc. as a global function.